e2e/npm_translate_lock_partial_clone/root/pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+    - '.'
+    - '../project-a'
```

Comment:
Was the lockfile generated with a `project-a/package.json` existing? Without that file if someone runs `pnpm install` will the lockfile change?

Reply:
Yes. If someone runs `pnpm install` in this project the lockfile would change and it would break the test case.

Reply:
I just wonder if we'll break/remove the original intention of the test when that happens, such as when renovate tries to update something